File: src/androidTest/java/pt/joaomneto/titancompanion/TestSOB.java
package pt.joaomneto.titancompanion;


import android.support.test.filters.LargeTest;
import android.support.test.runner.AndroidJUnit4;

import org.junit.runner.RunWith;

import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook;

import static pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.SEAS_OF_BLOOD;

@LargeTest
@RunWith(AndroidJUnit4.class)
public class TestSOB extends TestST {

    @Override
    protected FightingFantasyGamebook getGamebook() {
        return SEAS_OF_BLOOD;
    }


}

File: src/androidTest/java/pt/joaomneto/titancompanion/TestSOB.kt
package pt.joaomneto.titancompanion

import androidx.test.ext.junit.runners.AndroidJUnit4
import androidx.test.filters.LargeTest
import org.junit.runner.RunWith
import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.SEAS_OF_BLOOD

@LargeTest
@RunWith(AndroidJUnit4::class)
class TestSOB : TestST() {

    override val gamebook = SEAS_OF_BLOOD
}
